script.video.randomtv/addon.py

```diff
@@ -167,11 +177,33 @@ def onPlayBackStopped(self):
 
 	# Start Player
 	player.play(item=myPlaylist)
-	xbmc.sleep(100)
 #
 
 
-while (not xbmc.Monitor().abortRequested()):
+while (not xbmc.Monitor().waitForAbort(0.1)):
```

Comment:
I'll try higher.  I know I tried 1 second in the last version that used xbmc.sleep at the end of the loop and it was a too laggy for my taste.